README.md

```diff
@@ -25,7 +25,244 @@ To contribute, see the [contributing readme](CONTRIBUTING.md).
 
 ***
 
-# Using PivotalUI on your project
+# Using Pivotal UI on your project (with React)
+
+If you're ready to try PUI with React, follow these instructions!
+
+**NB** - We're assuming that you have the following setup on your project:
+
+  - **Browserify or Webpack** - Our React modules follow the CommonJS module
+    pattern. You will need to use [Browserify](http://browserify.org/) or
+    [Webpack](http://webpack.github.io/) to compile your javascript for use
+    in the browser.
+
+  - **A JSX transpiler** - It's easiest to write React code with [JSX](https://facebook.github.io/react/docs/jsx-in-depth.html).
+    You will need a transpiler to convert your JSX code into plain javascript
+    for use in the browser. We recommend [Babel](https://babeljs.io/).
+
+  - If these instructions don't make sense to you, don't worry! We're working
+    on a sample project template that we'll publish soon :grin:
+
+***
+
+1. Install React, if you haven't already
+
+   ```
+   npm install --save-dev react
+   ```
+
+1. Install a PUI module for the components you need.
+
+   ```
+   npm install --save pui-react-typography
+   ```
+
+   Our React component modules include all necessary CSS. So no need to install
+   additional CSS packages!
+
+1. Install and run Dr. Frankenstyle.
+
+   [Dr. Frankenstyle](http://github.com/pivotal-cf/dr-frankenstyle)
+   is a tool that compiles all the CSS required by your React components into
+   a single file. The simplist way to set it up is:
+
+   ```
+   npm install --save-dev dr-frankenstyle
+   dr-frankenstyle <path-to-your-asset-build-folder>
+   ```
+
+   The compiled CSS file (and all related fonts/images/etc.) are written to
+   the specified folder.
```

Comment:
Should we explain the CSS module dependency structure here? I don't want the docs to be too dense (and that info is in the Dr. F docs already), but I'm worried that the CSS stuff might seem a bit too automagical here.